frontend/package.json

```
@@ -22,6 +22,7 @@
     "@massalabs/prettier-config-as": "^0.0.2",
     "@massalabs/react-ui-kit": "0.0.4-dev.20240429153446",
     "@massalabs/wallet-provider": "^2.0.1-dev.20240423112601",
```

```suggestion
    "@massalabs/react-ui-kit": "^0.0.5-dev",
    "@massalabs/wallet-provider": "^2.0.1-dev",
```

then run npm i to update lock file